server/src/document-symbol.ts

```diff
@@ -40,7 +48,7 @@ export function collectDocumentSymbols(parent: tsp.NavigationTree, symbols: lsp.
 }
 
 export function collectSymbolInformations(uri: string, current: tsp.NavigationTree, symbols: lsp.SymbolInformation[], containerName?: string): boolean {
```

Comment:
We should probably do the same here for clients who cannot handle hierarchical symbols, can be a separate PR.